client/src/schemas/material.js

```diff
@@ -0,0 +1,156 @@
+import { object, string, array, boolean } from 'yup'
+
+export const categories = [
+  'CELL_LINE',
+  'PLASMID',
+  'PROTOCOL',
+  'DATASET',
+  'MODEL_ORGANISM',
+  'PDX',
+  'OTHER'
+]
+
+export const importSources = [
+  'GEO',
+  'SRA',
+  'DBGAP',
+  'DATASET',
+  'PROTOCOLS_IO',
+  'ADDGENE',
+  'JACKSON_LAB',
```

Comment:
In readable names this is `JACKSON_LABS`. Should they match?

Reply:
good catch my man